src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(number) {
+      calculator.result += number;
```

Comment:
use this

Reply:
i cannot use 'this' here. because of using 'operate' method this returns undefined.